credsweeper/file_handler/data_content_provider.py

```diff
@@ -53,6 +54,19 @@ def represent_as_structure(self) -> bool:
             text = self.data.decode(encoding='utf-8', errors='strict')
         except Exception:
             return False
+        # # # Python
+        try:
+            if ";" in text or 2 < text.count("\n"):
```

Comment:
I don't know why you checking `";"` in here.
Python doesn't use semicolon isn't it?

Reply:
Semicolon may be used in python to merge source to one line. 
It is a marker that there is something complicated than HelloWorld. Imports must be separated with semicolon or new lines.
```
import urllib.request; print(urllib.request.urlopen("https://github.com/").read())
```

Reply:
Thank you for explanation.